docs/conf.py

```diff
@@ -62,9 +62,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.0'
+version = '1.1'
 # The full version, including alpha/beta/rc tags.
-release = '1.0.0'
+release = '1.1.0'
```

Comment:
We should take a look at using ``bumpversion``. I started using it for a couple projects, it's quite nice and doesn't add any deps to your package.